.github/workflows/ci.yml

```diff
@@ -253,9 +253,7 @@ jobs:
           path: ~/.gradle/caches
           key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle') }}
           restore-keys: ${{ runner.os }}-gradle
-      - name: Build Node
-        run: ./gradlew build
```

Comment:
This was also running all standard tests, which is a waste / duplication of CI resources.

Reply:
Instead, `./gradlew :core:assemble` only runs the build and not also the tests. (Thanks gradle for being confusing like that).